src/admiral/certs/tasks.py

```diff
@@ -28,63 +28,53 @@
     retry_backoff=True,
     retry_jitter=True,
     retry_kwargs={"max_retries": 16},
+    rate_limit="10/h",
 )
-def summary_by_domain(domain, subdomains=True, expired=False):
+def summary_by_domain(domain, subdomains=True):
     """Fetch a summary of the certificates in the log.
 
     Arguments:
     domain -- the domain to query
     subdomains -- include certificates of subdomains
-    expired -- include expired certificates
 
     """
     # validate input
     m = DOMAIN_NAME_RE.match(domain)
     if m is None:
         raise ValueError(f"invalid domain name format: {domain}")
 
-    wildcard_param = "%." if subdomains else ""
-    expired_param = "" if expired else "&exclude=expired"
+    # fetch api key
+    key = ""
+    with open("/run/secrets/sslmate-api-key.txt", "r") as file:
+        key = file.read().rstrip()
 
-    logger.info(f"Fetching certs from CT log for: {wildcard_param}{domain}")
+    logger.info(f"Fetching certs from CT log for: {domain}")
     url = (
-        f"https://crt.sh/?Identity={wildcard_param}{domain}{expired_param}"
-        f"&output=json"
+        f"https://api.certspotter.com/v1/issuances?domain={domain}&include_subdomains={subdomains}"
+        f"&expand=dns_names&expand=cert"
     )
     req = requests.get(
         url,
-        headers={"User-Agent": "cyhy/2.0.0"},
+        headers={"Authorization": f"Bearer {key}", "User-Agent": "cyhy/2.0.0"},
```

Comment:
Do we want the user-agent to be something more specific than `cyhy`?  Maybe something like `admiral/1.2.3` would be better, since we can automatically update the string with the appropriate version.  It isn't clear to me where the `2.0.0` version for `cyhy` is coming from.

Reply:
I think the `cyhy/2.0.0` version was based on this being something of a demo for an envisioned cyhy v2 rework from long in the past.

Reply:
See [f01a4a9](https://github.com/cisagov/admiral/pull/31/commits/f01a4a9387878a44186ade1ee09da43e17484935).

Reply:
Nice!